src/conversion-settings.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as estree from 'estree';
+import {Iterable as IterableX} from 'ix';
+import * as jsc from 'jscodeshift';
+import {Analysis} from 'polymer-analyzer';
+
+import {getNamespaces} from './util';
+
+/**
+ * These are the settings used to configure the conversion. It contains
+ * important information about what should be analyzed, what should be ignored,
+ * and how each file should be formatted for conversion.
+ */
+export interface ConversionSettings {
+  /**
+   * Namespace names used to detect exports.
+   */
+  readonly namespaces: ReadonlySet<string>;
+  /**
+   * Files to exclude from conversion (ie lib/utils/boot.html).
+   */
+  readonly excludes: ReadonlySet<string>;
+  /**
+   * Files to exclusively include in conversion. Defaults to all files HTML
```

Comment:
Remove "exclusively" if these includes are additive to "all files HTML imported somewhere in the project".  It reads like this replaces whatever is imported somewhere.